test/basedriver/helpers-e2e-specs.js

```diff
@@ -86,8 +86,10 @@ describe('app download and configuration', function () {
             // for testing zip file content types
             if (req.url.indexOf('mime-zip') !== -1) {
               res.setHeader('content-type', 'application/zip');
-            } else if (req.url.indexOf('mime-bip') !== 1) {
+            } else if (req.url.indexOf('mime-bip') !== -1) {
               res.setHeader('content-type', 'application/bip');
+            } else if (req.url.indexOf('mime-parameter') !== -1) {
```

Comment:
I think `includes` would be more readable here in comparison to indexOf

Reply:
I refactored the whole thing by passing the actual header value as a query parameter.